ADALiOS/ADALiOS/ADProfileInfo.m

```diff
@@ -156,7 +156,7 @@ - (id)initWithEncodedString:(NSString*)encodedString
         AD_LOG_WARN(@"The id_token type is missing.", @"Assuming JWT type.");
     }
     
-    if (![allClaims objectForKey:PROFILE_INFO_PREFERRED_USERNAME])
+    if (![allClaims objectForKey:PROFILE_INFO_TENANTID])
```

Comment:
Will update this to accept oid or fail. The old will need to be past by the B2C admin for any mobile scenarios to work. Discussed with @skwan and @dstrockis